api/teams.ts

```diff
@@ -7,3 +7,7 @@ export const teamSecrets = {
   update: client("/teams/{id}/secrets/{name}").patch,
   delete: client("/teams/{id}/secrets/{name}").delete,
 };
+
+export const teamUsers = {
+  list: client("/teams/{teamId}/users").get,
+};
```

Comment:
based on this [PR](https://github.internal.digitalocean.com/paperspace/ps-api/pull/194/files) should also update and delete be added? or is that out of scope?